samples/snippets/dataset_access_permissions_test.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2021 Google LLC
```

Comment:
Nit: In the filename, "access" and "permissions" are redundant. Let's call it `dataset_access_test.py`.

Reply:
Done in latest commit!